apps/recipes-react-components/package.json

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31011&commitId=eb61eb7b49fcfbb4ddf48986ee0d544e425e5b41&env=prod&clientType=FLUENTUI)


  <details><summary><b>Table flex - subtle selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|27| Changed|
|Table flex - subtle selection.rest.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout flex</b> 43 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|27| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|31| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|24| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|25| Changed|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|25| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|26| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|31| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|25| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|31| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|27| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|21| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|27| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|27| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|24| Changed|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|26| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|27| Changed|
|Table layout flex.multiselect - RTL.chromium.png|27| Changed|
|Table layout flex.multiselect - Dark Mode.chromium.png|26| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|31| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|31| Changed|
|Table layout flex.multiselect.chromium.png|27| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|26| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|26| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|27| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|27| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|31| Changed|
|Table layout flex.primary cell.chromium.png|33| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|27| Changed|
|Table layout flex.single select (checked).chromium.png|27| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|31| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|27| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|26| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|27| Changed|
|Table layout flex.single select.chromium.png|27| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|31| Changed|
|Table layout flex.size - extra small.chromium.png|14| Changed|
|Table layout flex.single select - RTL.chromium.png|27| Changed|
|Table layout flex.single select - High Contrast.chromium.png|31| Changed|
|Table layout flex.single select - Dark Mode.chromium.png|26| Changed|
|Table layout flex.size - medium - Dark Mode.chromium.png|26| Changed|
|Table layout flex.size - medium - RTL.chromium.png|27| Changed|
|Table layout flex.size - medium.chromium.png|27| Changed|
|Table layout flex.size - small.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout flex - cell actions</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|31| Changed|
|Table layout flex - cell actions.always visible - RTL.hover row.chromium.png|27| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.chromium.png|26| Changed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|26| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|31| Changed|
|Table layout flex - cell actions.default - RTL.chromium.png|27| Changed|
|Table layout flex - cell actions.default - RTL.hover row.chromium.png|27| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|26| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|31| Changed|
|Table layout flex - cell actions.always visible.chromium.png|27| Changed|
|Table layout flex - cell actions.always visible - RTL.chromium.png|27| Changed|
|Table layout flex - cell actions.default - Dark Mode.chromium.png|26| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|31| Changed|
|Table layout flex - cell actions.always visible.hover row.chromium.png|27| Changed|
|Table layout flex - cell actions.default.hover row.chromium.png|27| Changed|
|Table layout flex - cell actions.default.chromium.png|27| Changed|
|Table layout flex - cell actions.in header cell.chromium.png|27| Changed|
|Table layout flex - cell actions.in header cell.hover row.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout flex - headers</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - headers.sortable - Dark Mode.hover header.chromium.png|26| Changed|
|Table layout flex - headers.sortable - Dark Mode.chromium.png|26| Changed|
|Table layout flex - headers.sortable - Dark Mode.press header.chromium.png|26| Changed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|31| Changed|
|Table layout flex - headers.sortable.press header.chromium.png|27| Changed|
|Table layout flex - headers.sortable.hover header.chromium.png|27| Changed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|31| Changed|
|Table layout flex - headers.sortable.chromium.png|27| Changed|
|Table layout flex - headers.sortable - High Contrast.chromium.png|31| Changed|

</details>


  <details><summary><b>Table layout flex - resizable columns</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - resizable columns.default.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout flex - truncate</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - truncate.default (disabled).chromium.png|27| Changed|
|Table layout flex - truncate.true.chromium.png|27| Changed|
|Table layout flex - truncate.false.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout table</b> 43 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|27| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|31| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|24| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|25| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|25| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|24| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|21| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|26| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|31| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|25| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|27| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|26| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|27| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|27| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|27| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|31| Changed|
|Table layout table.multiselect - RTL.chromium.png|27| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|26| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|31| Changed|
|Table layout table.multiselect.chromium.png|27| Changed|
|Table layout table.primary cell.chromium.png|33| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|26| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|27| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|26| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|31| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|31| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|27| Changed|
|Table layout table.single select (checked).chromium.png|27| Changed|
|Table layout table.single select (checked) - brand.chromium.png|27| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|27| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|31| Changed|
|Table layout table.single select - Dark Mode.chromium.png|26| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|27| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|26| Changed|
|Table layout table.single select - High Contrast.chromium.png|31| Changed|
|Table layout table.single select.chromium.png|27| Changed|
|Table layout table.size - extra small.chromium.png|14| Changed|
|Table layout table.size - medium - Dark Mode.chromium.png|26| Changed|
|Table layout table.size - medium - RTL.chromium.png|27| Changed|
|Table layout table.single select - RTL.chromium.png|27| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|31| Changed|
|Table layout table.size - medium.chromium.png|27| Changed|
|Table layout table.size - small.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout table - cell actions</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|26| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|31| Changed|
|Table layout table - cell actions.always visible.chromium.png|27| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|31| Changed|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|27| Changed|
|Table layout table - cell actions.always visible.hover row.chromium.png|27| Changed|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|26| Changed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|26| Changed|
|Table layout table - cell actions.default - Dark Mode.chromium.png|26| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|31| Changed|
|Table layout table - cell actions.always visible - RTL.chromium.png|27| Changed|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|27| Changed|
|Table layout table - cell actions.default - RTL.chromium.png|27| Changed|
|Table layout table - cell actions.default.chromium.png|27| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|31| Changed|
|Table layout table - cell actions.default.hover row.chromium.png|27| Changed|
|Table layout table - cell actions.in header cell.chromium.png|27| Changed|
|Table layout table - cell actions.in header cell.hover row.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout table - headers</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|26| Changed|
|Table layout table - headers.sortable - Dark Mode.chromium.png|26| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|31| Changed|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|26| Changed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|31| Changed|
|Table layout table - headers.sortable.chromium.png|27| Changed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|31| Changed|
|Table layout table - headers.sortable.press header.chromium.png|27| Changed|
|Table layout table - headers.sortable.hover header.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout table - resizable columns</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - resizable columns.default.chromium.png|27| Changed|

</details>


  <details><summary><b>Table layout table - truncate</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout table - truncate.true.chromium.png|27| Changed|
|Table layout table - truncate.default (disabled).chromium.png|27| Changed|
|Table layout table - truncate.false.chromium.png|27| Changed|

</details>


  <details><summary><b>Table table - subtle selection</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table table - subtle selection.rest.hover unselected row.chromium.png|27| Changed|
|Table table - subtle selection.rest.chromium.png|27| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>